keyboards/beekeeb/3w6hs/info.json

```diff
@@ -0,0 +1,71 @@
+{
+    "manufacturer": "beekeeb",
+    "keyboard_name": "3w6hs",
+    "maintainer": "beekeeb",
+    "diode_direction": "COL2ROW",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0003",
+        "vid": "0xBEEB",
+        "no_startup_check": true
+    },
+    "matrix_pins": {
+        "custom_lite": true
+    },
```

Comment:
Not sure why, but moving this back to rules.mk fixes the errors...

Reply:
I've set both `custom_lite` and `custom` as `true` to make it work. Not sure if it's the right way though.

Reply:
Nope, not the right way, custom=true means full matrix replacement.